FluentConstraints.podspec

```diff
@@ -1,13 +0,0 @@
-Pod::Spec.new do |s|
```

Comment:
I don't think you want to delete this file. It's what allows you to upload a new pod of FluentConstraints itself to CocoaPods.

Remember to bump the version (probably to 2.0) if you restore this file.